spec/rubocop/cop/rspec/repeated_block_body_spec.rb

```diff
@@ -0,0 +1,57 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RepeatedBlockBody do
+  subject(:cop) { described_class.new }
+
+  it 'registers an offense for repeated describe body' do
+    expect_offense(<<-RUBY)
+      describe 'doing x' do
+      ^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
+        it { cool_predicate_method }
+      end
+
+      describe 'doing y' do
+      ^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
+        it { cool_predicate_method }
+      end
+    RUBY
+  end
+
+  it 'registers an offense for repeated context body' do
+    expect_offense(<<-RUBY)
+      context 'when awesome case' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
+        it { cool_predicate_method }
+      end
+
+      context 'when another awesome case' do
+      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Don't repeat block body
+        it { cool_predicate_method }
+      end
+    RUBY
+  end
```

Comment:
What if one of them is a `context` and another is `describe`?

Reply:
I have added that case and there should not be the offense. But I'm not sure if it is the right behavior